bat/tests/09-no-delta-packs-over-format-bump/run.bats

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bats
+
+# shared test functions
+load ../../lib/mixerlib
+
+setup() {
+  global_setup
+}
+
+# TODO: just call the auto-format-bump capability when available. Not using the old
+# auto-format-bump functionality because it will be changing very soon.
+
+@test "Attempt to create delta packs over format bump" {
+  mixer-init-stripped-down $CLRVER 10
+  sed -i 's/\(FORMAT\).*/\1 = "1"/' mixer.state
+
+  ###############################################################################
+  # +0
+  ###############################################################################
+
+  # build bundles and updates regularly
+  mixer-build-bundles > $LOGDIR/build_bundles10.log
+  mixer-build-update > $LOGDIR/build_update10.log
+
+  ###############################################################################
+  # +10
+  ###############################################################################
+
+  # update mixer to build version 20, which in our case is the +10
+  mixer-versions-update 20
+  # build bundles normally. At this point the bundles to be deleted should still
+  # be part of the mixbundles list and the groups.ini
+  mixer-build-bundles > $LOGDIR/build_bundles20.log
+  # no deleted bundles
+
+  # Replace the +10 version in /usr/lib/os-release with +20 version and write the
+  # new format to the format file on disk.  This is so clients will already be on
+  # the new format when they update to the +10 because the content is the same as
+  # the +20.
+  sudo sed -i 's/\(VERSION_ID=\).*/\130/' update/image/20/full/usr/lib/os-release
+  echo 2 | sudo tee update/image/20/full/usr/share/defaults/swupd/format
+  # build update based on the modified bundle information. This is *not* a
+  # minversion and these manifests must be built with the mixer from the original
+  # format (if manifest format changes).
+  mixer-build-update > $LOGDIR/build_update20.log
+
+  # not validating the bump itself, we have another test for that
+
+  ###############################################################################
+  # +20
+  ###############################################################################
+
+  # update mixer to build version 30, which in our case is the +20
+  mixer-versions-update 30
+  # update mixer.state to new format
+  sudo sed -i 's/\(FORMAT\).*/\1 = "2"/' mixer.state
```

Comment:
Optional: to better future proof the regex, consider using `^\(FORMAT\) =.*` instead.